docs/tutorials.md

```diff
@@ -0,0 +1,59 @@
+# Tutorials
+
+In addition to the basic getting started guides, we offer tutorials that aim to
+show how Material for MkDocs can be used in different use cases. In contrast to
+the getting started guides or the reference documentation, the tutorials show
+the breadth of functionality available with Material for MkDocs but also within
+the wider MkDocs ecosystem.
+
+* [Blogs](tutorials/blogs.md)
+
+## Tutorial Structure
+
+The tutorials guide you through worked examples, so following them you should
+gain not only and understanding of how Material for MkDocs can be used but also
+a template for your own projects. For convenience, these templates are also
+available as [template repositories on GitHub].
+
+<!-- TODO -->
+[template repositories on GitHub]: templates.md
+
+Each tutorial is a single page, so use the Table of Contents navigation to move
+between numbered steps. They start by mentioning prerequisites and giving an
+overview of the content as well as describing the outcomes.
+
+### Instructions
+
+Tutorials contain a mix of explanation, demonstrations, and instructions that you
+should follow as you work your way through the individual steps. Those instructions
+are shown in the form of admonitions to separate them from the rest of the
+tutorial:
+
+!!! example "Instructions to follow"
+
+    Whenever you see a box  like this you know it is time to get hands on and
+    apply material just covered as well as reflect on how you would it in your
+    work work.
```

Comment:
This line is reading a bit weird. Especially the "work work" which must be a typo.

Reply:
Jup, as I mentioned in https://github.com/squidfunk/mkdocs-material/pull/7014#discussion_r1554825355, this is a draft and not proof-read.